airflow/api_internal/internal_api_call.py

```diff
@@ -103,12 +110,21 @@ def internal_api_call(func: Callable[PS, RT]) -> Callable[PS, RT]:
     See [AIP-44](https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-44+Airflow+Internal+API)
     for more information .
     """
+
     from requests.exceptions import ConnectionError
 
+    def is_retryable_exception(exception: BaseException) -> bool:
```

Comment:
This is a piurely internal method, can you add some explanaition "why" and mark it as private?
```suggestion
    def _is_retryable_exception(exception: BaseException) -> bool:
    """
    Evaluates which exception types should be re-tried.
    
    This is especially demanded for cases where (a) an application gateway or Kubernetes ingress can
    not find a running instance of a webserver hostring the API (HTTP 502+504) or when the
    HTTP request fails in general on network level.
    
    Note that we want to fail on other general errors on the webserver not to send bad requests in an endless loop.
    """
```